### DISCUSSION TITLE
Rustdesk free app won't connect outside lan

### DISCUSSION THREAD

**Discussion by MoHara23** [rustdesk/rustdesk#5755] [Q&A]
Rustdesk 1.2.2   version date 2023-08-22 01:58 on Windows 10 , no self host server in use,  ports 21115-21117 TCP and 21116 UDP are open (checked with PFPortchecker app).

Yet when I switch from wifi to 4G on my phone, I don't get connected to the Win 10 laptop.   In LAN connection is ok.

error message if not on LAN:

irst:
broken pipe (os error 32)
then:
reset by the peer 
then repeats

Did i mis some other setting?

Please help, I spend a lot of time and my  head... ? It's also my first time using and knowing about Rustdesk.

**Comment by dinger1986** [comment]
Can you try it on another device?

Maybe an issue with ipv6?

**Reply by MoHara23** [reply]
From laptop outside LAN, worked.

How come not from Iphone?
it's working for LAN great.

ip adres has format  xx.xxx.xxx.xxx and changes often

**Comment by MoHara23** [comment]
Myip site shows a ip4 adres so no ipv6. I will give it a try to test it on another laptop that is at my brothers house.

**Comment by MoHara23** [comment]
Is it possible that portforward is not needed? i tried another latop to acces that had not yet any port forward configured to that laptop in my lan, yet I could connect to it , from outside LAN.

So I can delete the rules i made for portforward for the other laptop?

**Reply by dinger1986** [reply]
No you don't need the port forwards.

I don't know why it's not working on your phone.

**Comment by MoHara23** [comment]
Does someone can test if it works on 4G 5 G  on iphone? if it works then I need to conact my mobile phone provider.

I really need acces with my phone.

**Comment by MoHara23** [comment]
Dear, update about 4G, when I connect to vpn (nord vpn) on my Iphone while 4G is active and Wifi disabled, then Rustdesk connect to the laptop with ease.   A good work arrround for now.
